main.js

```diff
@@ -133,3 +143,14 @@ function displayCard() {
     </article>`
 }
 
+function createNewActivity() {
+  addClass(logButton)
+  addClass(startNewActivityForm)
+  removeClass(defaultForm)
+  resetButtons()
+  startButton.disabled = false
```

Comment:
Might want to consider using a function to toggle this as we update the disabled property a few times